cmd/opm/alpha/template/composite.go

```diff
@@ -31,11 +34,23 @@ and a 'composite template' file`,
 		Args: cobra.MaximumNArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
 			containerTool = "docker"
-			catalogData, err := os.Open(catalogFile)
+			var tempCatalog io.ReadCloser
+			catalogURI, err := url.ParseRequestURI(catalogFile)
 			if err != nil {
-				log.Fatalf("opening catalog config file %q: %s", catalogFile, err)
+				tempCatalog, err = os.Open(catalogFile)
+				if err != nil {
+					log.Fatalf("opening catalog config file %q: %s", catalogFile, err)
+				}
+				defer tempCatalog.Close()
+			} else {
+				tempResp, err := http.Get(catalogURI.String())
+				if err != nil {
+					log.Fatalf("fetching remote catalog config file %q: %s", catalogFile, err)
```

Comment:
Unrelated to this PR, per se, but one of the confusing bits of nomenclature we have in this space is the names that we use to designate the spec/files from the operator author and catalog maintainer which fulfill the roles of this conversation.  `catalog/config` and `catalogs.yaml` are too similar.  
We're effectively canonicalizing `catalog config file` here, which I think is fine, so maybe we start referring to the operator author's file as `contributions spec/file` WDYT?

Reply:
That sounds reasonable to me

Reply:
/resolve